modules/rhel-attaching-instance-aws.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * machine_management/adding-rhel-compute.adoc
+// * machine_management/more-rhel-compute.adoc
+
+
+[id="rhel-attaching-instance-aws_{context}"]
+= Attaching the role permissions to {op-system-base} instance in AWS
+
+Using the Amazon IAM console in your browser, you may select the needed roles and assign them to a worker node.
+
+.Procedure
+. From the AWS IAM console, create your link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#create-iam-role[desired IAM role].
+. After the role has been created, link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#attach-iam-role[attach the IAM role] to the desired worker node.
```

Comment:
I don't think that "After the role has been created, " is necessary here, because since this is a procedure, we assume that you performed step 1 before you got to this step 2.